.github/workflows/sagemaker-integration.yml

```diff
@@ -6,7 +6,7 @@ on:
       mode:
         description: "release/nightly containers to test. Default is nightly"
         required: false
-        default: nightly
+        default: 'nightly'
```

Comment:
the true default will be empty (if triggered by scheduler) please make sure you have this checked in the script.